vendor/tracing/zero/destinations.ml

```diff
@@ -32,7 +32,69 @@ let direct_file_destination ?(buffer_size = 4096 * 16) ~filename () =
   (module Dest : Writer_intf.Destination)
 ;;
 
-let file_destination ~filename () = direct_file_destination ~filename ()
+let compressed_file_destination ?(buffer_size = 4096 * 16) ~filename () =
+  let buf = Iobuf.create ~len:buffer_size in
+  let com_size = buffer_size * 2 in
+  let com_level = 5 in
+  let compressed_buf = Iobuf.create ~len:com_size in
+  let file = Core_unix.openfile ~mode:[ O_CREAT; O_TRUNC; O_RDWR ] filename in
```

Comment:
```suggestion
  let file = Core_unix.openfile ~mode:[ O_CREAT; O_TRUNC; O_CLOEXEC; O_RDWR ] filename in
```

In case this ends up being used with a trace processor shell-based `-serve`.

Reply:
Sorry, I think this review got accidentally marked as resolved.

Reply:
Just added the O_CLOEXEC in direct_file_destination as well. Does that resolve this review?